auth.ts

```diff
@@ -11,7 +11,7 @@ export class BasicCredentialHandler implements ifm.IRequestHandler {
     }
 
     prepareRequest(options:any): void {
-        options.headers['Authorization'] = 'Basic ' + new Buffer(this.username + ':' + this.password).toString('base64');
```

Comment:
good

Reply:
Using `new Buffer` is deprecated, this is equivalent. See https://nodejs.org/api/buffer.html#buffer_new_buffer_array

Reply:
Yep, as I point out above, this was a good change.